src/com/twitter/intellij/pants/service/project/resolver/PantsSourceRootsExtension.java

```diff
@@ -133,7 +133,7 @@ private void installSyntheticModules(
         PantsConstants.SYSTEM_ID,
         ModuleTypeId.JAVA_MODULE,
         moduleName,
-        targetPath,
+        projectDataNode.getData().getIdeProjectFileDirectoryPath() + "/" + moduleName,
```

Comment:
Will that be okay with platforms that use a different file separator?

Reply:
I'm not sure, but we already have three expressions like this https://github.com/pantsbuild/intellij-pants-plugin/search?q=getIdeProjectFileDirectoryPath&unscoped_q=getIdeProjectFileDirectoryPath

Reply:
yeah, file.separator is more hygienic, but since we are only support linux and mac, it works for now.